kore/src/Kore/Step/Rule.hs

```diff
@@ -360,10 +359,28 @@ onePathRuleToPattern (OnePathRule rulePatt) =
             (Predicate.unwrapPredicate . requires $ rulePatt)
             (left rulePatt)
         )
-        ( mkAnd
-            (Predicate.unwrapPredicate . ensures $ rulePatt)
-            (right rulePatt)
-        )
+       ( mkApplyAlias
+            wEF
+            [( mkAnd
+                (Predicate.unwrapPredicate . ensures $ rulePatt)
+                (right rulePatt)
+            )]
+       )
+  where
+    wEF :: Alias
```

Comment:
Because this definition is special, maybe we should promote it to the top-level.